src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateValue.txt

```diff
@@ -261,4 +261,7 @@ Date(2045,2,3)
 Date(45,2,3)
 
 >> DateValue("02 03, 0045 04:56:07")
-Date(45,2,3)
\ No newline at end of file
+Date(45,2,3)
+
+>> DateValue("10/1/1920","vi-vn")
```

Comment:
Can we have more tests to make sure it works, like the string in different format 10-1-1920 or 1920/01/10 or so on.